packages/headless/src/features/search/search-request.ts

```diff
@@ -116,19 +116,28 @@ function getAllFacets(state: StateNeededBySearchRequest) {
 
 function getSpecificFacetRequests<T extends FacetSetState>(state: T) {
   return getFacetRequests(state).map((request) => {
-    /* The Search API does not support 'alphanumericDescending' as a string value and instead relies on a new sort mechanism to specify sort order.
-    At the moment, this is only supported for alphanumeric sorting, but will likely transition to this pattern for other types in the future. */
-    if (request.sortCriteria === 'alphanumericDescending') {
-      return {
-        ...request,
-        sortCriteria: {
-          type: 'alphanumeric',
-          order: 'descending',
-        },
+    let newSortCriteria = request.sortCriteria;
+    if (
```

Comment:
Uh funky.
Nothing came to mind tho this looks like a fitting hack to me 👍

Reply:
> Uh funky.

Yup, agreed 🙃 I'll reach out to the Search API team and ask if there's anyway they could support `automatic` within the new type so we wouldn't need this hack. Surely it would be simple enough to handle `automatic` the same as it currently works for the new type...

Reply:
UPDATE: I spent some time this morning trying to implement a more "concrete" solution where it's enforced by type (e.g., `automatic` and `alphanumericDescending` aren't valid for `type` within `SpecificSortCriteriaExplicit`), but it introduced more edge-case handling throughout... 

That being said, I'm still trying to get a hold of someone from the Search API team to figure out exactly what their future plans are. This works now, but if it turns out that we will need to support both string (`automatic`) and object values simultaneously for the foreseeable future, I think it's worth investing the time to implement something that's less of a hack 🤔